src/modules/missionHelper/i18n/de_DE.json

```diff
@@ -3,6 +3,7 @@
     "expansions": "Ausbreitung | Ausbreitungen",
     "followups": "Folgeeinsatz | Folgeeinsätze",
     "generated_by": "Generiert von",
+    "ignore_mission_IDs": [143,613],
```

Comment:
we could use the root `transfer_missions` translation instead. That would reduce redundant data.